core/frontend/src/tile/IDBTileStorage.ts

```diff
@@ -0,0 +1,182 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Tiles
+ */
+
+import { TileRequestChannel } from "./internal";
+
+export class IDBStorage {
```

Comment:
IDBTileStorage has been changed to IDBStorage, a more generic class that has no mention of tiles. All of its functionality deals solely with the IndexedDB database itself.